src/cmd/MVP-4/.env

```diff
@@ -1,19 +1,19 @@
-CREDENTIALS=".\\secret.json"
+CREDENTIALS="./secret.json"
```

Comment:
Does this work on windows?

Reply:
I don't think so, making this compatible with multiple OS will be tricky